app/trail-document-screen.tsx

```
@@ -1,58 +1,291 @@
-import HomeHeader from "@/components/ui/header";
-import TrailEventHeader from "@/components/ui/trail-event-header";
-import type { Event } from "@/services/event-service";
-import { getEventById } from "@/services/event-service";
-import { useLocalSearchParams, useRouter } from "expo-router";
+import BottomNav from "@/components/ui/bottom-nav";
+import Header from "@/components/ui/header";
```

ESLint

[error] Parse errors in imported module '@/components/ui/header': Merge conflict marker encountered. (7:0)

(import/namespace)